ReleaseTooling/Sources/ZipBuilder/FrameworkBuilder.swift

```diff
@@ -639,33 +635,7 @@ struct FrameworkBuilder {
           "\(framework): \(error)")
       }
 
-      // Move any privacy manifest-containing resource bundles into the
```

Comment:
No functional change here. Just refactored to a function.

Reply:
LGTM. This helps with readability.